analyses/cms-open-data-ttbar/validate_histograms.py

```diff
@@ -42,8 +42,39 @@ def validate(histos: dict, reference: dict) -> dict[str, list[str]]:
         if not np.allclose(h['edges'], ref_h['edges']):
             errors[name].append(f"Edges do not match:\n\tgot      {h['edges']}\n\texpected {ref_h['edges']}")
         contents_depend_on_rng = "pt_res_up" in name # skip checking the contents of these histograms as they are not stable
+
         if not contents_depend_on_rng and not np.allclose(h['contents'], ref_h['contents']):
-            errors[name].append(f"Contents do not match:\n\tgot      {h['contents']}\n\texpected {ref_h['contents']}")
+
+            # check if bin migration
+            is_close = np.isclose(h['contents'], ref_h['contents'])
+            where_not_close = np.where(np.invert(is_close))
+            diff_values = np.diff(where_not_close)
+            split_indices = np.argwhere(np.abs(diff_values[0])>1)
```

Comment:
I think the `[0]` here logically belongs at line 50, where `np.where` returns an extra dimension that we already don't care about at line 51. removes the need to repeat the `[0]` at the lines below too.